### DISCUSSION TITLE
Does Robyn support group wise Ridge regression? Can there be two or three Group bys?

### DISCUSSION THREAD

**Discussion by CJ2407** [facebookexperimental/Robyn#298] [Q&A]
How do create a mix model using Roby for different Channels (Retail vs Ecomm) and different products categories(Accessories vs Clothing vs Footwears) for a retailer?

**Comment by gufengzhou** [comment]
Hi, I've answered a similar question here https://github.com/facebookexperimental/Robyn/discussions/286
